deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules

```diff
@@ -33,6 +33,16 @@ ACTION=="add", KERNEL=="sd*", SUBSYSTEMS=="usb", \
   ENV{ID_SERIAL}="USB-%c-$env{ID_INSTANCE}", \
   ENV{ID_SERIAL_SHORT}="%c"
 
+# JMicron JMS561U two ports SATA 6Gb/s bridge
+ACTION=="add", KERNEL=="sd*", SUBSYSTEM=="block", \
+  ENV{ID_VENDOR_ID}=="152d", ENV{ID_MODEL_ID}=="1561", \
+  PROGRAM="/usr/lib/udev/serial_id %N", \
```

Comment:
Does it work when you use `PROGRAM="serial_id %N", \` here?

Reply:
No, it does not. udevadm test /sys/block/sdX reports failure to execute serial_id without the explicit path to /usr/lib/udev/

Reply:
OK, good to know that. Thanks for testing this out.